sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/utils/ResourceManagerThrottlingInfo.java

```diff
@@ -47,6 +50,11 @@ public ResourceManagerThrottlingInfo(HttpHeaders headers) {
             }
         }
         resourceRateLimit = headers.getValue(RESOURCE_RATE_LIMIT_HEADERS);
+        Matcher matcher = Pattern.compile("\\w+\\.\\w+/([^;]+);(\\d+)").matcher(resourceRateLimit);
```

Comment:
Should we put `Pattern.compile("\\w+\\.\\w+/([^;]+);(\\d+)")` to a static, to save runtime cost to do it every time?

Reply:
done